code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -334,6 +342,10 @@
 		var/is_sillycone = issilicon(user)
 		if(!(bot_cover_flags & BOT_COVER_EMAGGED) && (is_sillycone || user.Adjacent(src)))
 			. += span_info("Alt-click [is_sillycone ? "" : "or use your ID on "]it to [bot_cover_flags & BOT_COVER_LOCKED ? "un" : ""]lock its control panel.")
+	if(paicard)
+		. += span_notice("It has a pAI device installed.")
+		if(!(bot_cover_flags & BOT_COVER_OPEN))
+			. += span_info("You can use a <b>hemostat</b> to remove it.")
```

Comment:
was it always a hemostat? i coulda sworn it was like a DVD drive where it whirred the paicard out, but i might be going senile

Reply:
likely out of scope if you just reverted the PR that changed this so you don't have to change it now but man it's silly that you need a surgical tool because we don't have any other equivalent for pliers

Reply:
There is also an eject button in the bot panel, the hemostat is for if you don't have that access I guess.

Reply:
wack.